models/t_srv.go

```diff
@@ -0,0 +1,64 @@
+package models
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// SetTargetSRV sets the SRV fields.
+func (rc *RecordConfig) SetTargetSRV(priority, weight, port uint16, target string) error {
```

Comment:
Are all of these variants actually used? I hate dead code, and would rather only add as we need to.

Reply:
I commented out all the SetTarget$RTYPE functions and only brought back the ones that are in use. They're all in use :-)